scripts/pre_commit_linter.py

```diff
@@ -1976,16 +1983,27 @@ def _check_comments(self):
             print 'Starting comment checks'
             print '----------------------------------------'
         summary_messages = []
+        # Exclude scripts/start.sh due to unique commenting in that file alone.
         files_to_check = [
             filepath for filepath in self.all_filepaths if not
             any(fnmatch.fnmatch(filepath, pattern) for pattern in
-                EXCLUDED_PATHS) and filepath.endswith('.py')]
+                EXCLUDED_PATHS) and not filepath.endswith('scripts/start.sh')
+            ]
         message = 'There should be a period at the end of the comment.'
         failed = False
-        space_regex = re.compile(r'^#[^\s].*$')
-        capital_regex = re.compile('^# [a-z][A-Za-z]* .*$')
         with _redirect_stdout(_TARGET_STDOUT):
             for filepath in files_to_check:
+                comment_start_chars = '#'
```

Comment:
This assumes that the file will be in python, bash, etc, which might not be the case. I would recommend wrapping this inside an `elif`.

Reply:
That being said, you might want put in logic that changes the check depending on whether these variables are defined.